app.drey.PaperPlane.json

```diff
@@ -0,0 +1,232 @@
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "rlottie",
+            "buildsystem": "meson",
+            "config-opts": [
+                "-Dbuildtype=release",
+                "-Dexample=false",
+                "-Dwerror=false"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/paper-plane-developers/rlottie",
+                    "commit": "1dd47cec7eb8e1f657f02dce9c497ae60f7cf8c5"
+                }
+            ]
+        },
+        {
+            "name": "blueprint-compiler",
+            "buildsystem": "meson",
+            "config-opts": [
+                "-Dbuildtype=release"
+            ],
+            "cleanup": [
+                "*"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://gitlab.gnome.org/jwestman/blueprint-compiler/-/archive/v0.8.1/blueprint-compiler-v0.8.1.tar.gz",
+                    "sha256": "9207697cfac6e87a3c0ccf463be1a95c3bd06aa017c966a7e352ad5bc486cf3c"
+                }
+            ]
+        },
+        {
+            "name": "libsass",
+            "buildsystem": "meson",
+            "cleanup": [
+                "*"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/lazka/libsass.git",
+                    "commit": "302397c0c8ae2d7ab02f45ea461c2c3d768f248e"
+                }
+            ]
+        },
+        {
+            "name": "sassc",
+            "buildsystem": "meson",
+            "cleanup": [
+                "*"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/lazka/sassc.git",
+                    "commit": "82803377c33247265d779af034eceb5949e78354"
+                }
+            ]
+        },
+        {
+            "name": "gtk",
+            "buildsystem": "meson",
+            "config-opts": [
+                "-Dbuildtype=release",
+                "-Dintrospection=disabled",
+                "-Dprint-cups=disabled",
+                "-Ddemos=false",
+                "-Dbuild-testsuite=false",
+                "-Dbuild-examples=false",
+                "-Dbuild-tests=false"
+            ],
+            "cleanup": [
+                "/bin/gtk*"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.gnome.org/GNOME/gtk.git",
+                    "commit": "06b3ced8e7fc936caed43379b120d75be09713ca",
+                    "tag": "4.10.3"
+                },
+                {
+                    "type": "patch",
+                    "path": "gtk-reversed-list.patch"
+                }
+            ]
+        },
+        {
+            "name": "appstream",
```

Comment:
btw if you use this to generate the release in the appstream file, you should make that part of the `dist` process that generate your tarball instead. Because that's were it belongs.

Not a blocker for flathub though.

Reply:
Appstream is used here rather as a runtime dependency for libadwaita (see https://gitlab.gnome.org/GNOME/libadwaita/-/commit/876672aac1191e4ac26d835cb75761ce07ef69ec).